<launch>

<arg name="qml_path" default="$(find d3_gui)/qml/kikiantoine.qml" />
<arg name="image_path" default="$(find d3_gui)/qml/robot.png" />
<arg name="flags_path" default="$(find d3_gui)/flags" />

<node name="d3_base_station_gui" pkg="d3_gui" type="base_station_gui">
	<param name="qmlPath" value="$(arg qml_path)"/>
	<param name="imagePath" value="$(arg image_path)"/>
	<param name="flagsPath" value="$(arg flags_path)"/>
</node>

</launch>